numba/core/ir_utils.py

```diff
@@ -2056,20 +2056,29 @@ def resolve_mod(mod):
         return None
 
 
+def enforce_no_dels(func_ir):
+    """
+    Enforce there being no ir.Del nodes in the IR.
+    """
+    strict = True
```

Comment:
remove this option or make it configurable

Reply:
Opt to remove. The pass manager is enforcing.